templates/docs/examples/base/empty-state/no-content-page-view.html

```diff
@@ -0,0 +1,26 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Empty State / No content - Page view format{% endblock %}
+
+{% block standalone_css %}base{% endblock %}
+
+{% block content %}
+<table aria-label="Vanilla framework empty table view format">
+    <thead>
+      <tr>
+        <th>Vendor ID</th>
+        <th>Product</th>
+        <th>ProductID</th>
+        <th>Driver</th>
+        <th>Huma Node</th>
+        <th>PCI Address</th>
+      </tr>
+    </thead>
+    <caption>
+        <div class="p-strip">
```

Comment:
I think the formatting is a bit off in the HTML, there should be only 1 indentation

Reply:
Fixed :+1: